### DISCUSSION TITLE
Xhttp节点如何设置

### DISCUSSION THREAD

**Discussion by ZhangSir9901** [2dust/v2rayNG#4132] [General]
1、新版去掉了规则预设值，使用非常不方便。路由设置没有之前的好用。请改进。谢谢

2、新增的xhttp模式如何配置，请给个示例。谢谢。

**Comment by lyrnya** [comment]
参考我的服务端[配置](https://github.com/lyrnya/config/blob/main/xray/VLESS-XHTTP-TLS.json)，v2rayng设置如下
![Screenshot_20241215-220734](https://github.com/user-attachments/assets/f266cdb9-3b76-4676-a6e4-35293794e907)
模式选择auto
path需要与服务端一致，如果没有设置就是"/"
不优选cdn只需要填写xhttp path，tls也可以留空不填
优选cdn则需要将服务器域名填写到xhttp host以及tls sni

另外alpn可选h3，不过我这里udp限速就没有选h3

**Reply by ZhangSir9901** [reply]
非常感谢。
